src/rgw/rgw_sal_motr.cc

```diff
@@ -2190,99 +2192,138 @@ int MotrObject::read_mobj(const DoutPrefixProvider* dpp, int64_t off, int64_t en
   return rc;
 }
 
-int MotrObject::get_bucket_dir_ent(const DoutPrefixProvider *dpp, rgw_bucket_dir_entry& ent)
+int MotrObject::get_index_key(const DoutPrefixProvider *dpp, std::string& prev_null_obj_key)
```

Comment:
This function is for fetching the reference to null object so should be named as `fetch_null_obj_reference`

Reply:
done